src/typing_extensions.py

```diff
@@ -1147,6 +1148,42 @@ def __repr__(self):
                                above.""")
 
 
+class _DefaultMixin:
+    """Mixin for TypeVarLike defaults."""
+
+    def __init__(self, default):
+        if isinstance(default, tuple):
```

Comment:
```suggestion
        if isinstance(default, (tuple, list)):
```
Whilst tuples are the internal ways these are stored mypy shows ParamSpec args as lists so I think that would be good to support